README.rst

```diff
@@ -208,9 +208,9 @@ To upload a file, pass either a file-like object or a tuple of
     >>> data = StringIO('StringIO instance has no .name attribute!')
     >>> from smartfile import BasicClient
     >>> api = BasicClient()
-    >>> api.post('/path/data/', file=('foobar.png', data))
-    >>> # Or use a file-like object with a name attribute
-    >>> api.post('/path/data/', file=file('foobar.png', 'rb'))
+    >>> f = ('foobar.png', data)
+    >>> api.upload(f)
+
```

Comment:
I think the example as it stands will not work because you need to set the position of the file object to `0`. It should be something like this (not tested):

```
      >>> from StringIO import StringIO
      >>> from smartfile import BasicClient           
      >>> f = StringIO('StringIO instance has no .name attribute!')
      >>> f.seek(0)
      >>> api = BasicClient()             
      >>> api.upload(f)
```